bin/openupgrade/openupgrade_log.py

```diff
@@ -1,5 +1,6 @@
 # -*- coding: utf-8 -*-
-from openupgrade_tools import table_exists
+from openupgradelib import openupgrade_tools
```

Comment:
That is what the next line does, by referring to openupgradelib.openupgrade_tools explicitely. Line 2 above is void here, please remove.